plasma_framework/contracts/poc/fast_exits/Liquidity.sol

```diff
@@ -0,0 +1,186 @@
+pragma solidity 0.5.11;
+pragma experimental ABIEncoderV2;
+
+import "../../src/exits/payment/PaymentExitGame.sol";
+import "../../src/framework/PlasmaFramework.sol";
+import "../../src/transactions/PaymentTransactionModel.sol";
+import "../../src/utils/PosLib.sol";
+import "../../src/framework/models/BlockModel.sol";
+import "../../src/utils/Merkle.sol";
+import "../../src/exits/payment/routers/PaymentStandardExitRouter.sol";
+
+/**
+ * @title Liquidity Contract
+ * Implementation Doc - https://github.com/omisego/research/blob/master/plasma/simple_fast_withdrawals.md
+*/
+contract Liquidity {
+    PaymentExitGame private paymentExitGame;
+
+    PlasmaFramework private plasmaFrameworkInstance;
+
+    mapping(address => uint160[]) public userExitIds;
+    mapping(uint160 => address) public exitIdtoUser;
+    mapping(uint160 => uint256) public exitIdtoAmount;
+
+    /**
+     * @notice provide PlasmaFramework contract-address when deploying the contract
+    */
+    constructor(address plasmaFrameworkContract) public {
+        plasmaFrameworkInstance = PlasmaFramework(plasmaFrameworkContract);
+        paymentExitGame = PaymentExitGame(plasmaFrameworkInstance.exitGames(1));
+    }
+
+    function getCurrentBondSize() public view returns (uint128) {
+        return paymentExitGame.startStandardExitBondSize();
+    }
+
+    /**
+     * @dev Call this func to start the exit on Rootchain contract
+     * @param utxoPosToExit position of the output which the contract has to exit
+     * @param rlpOutputTxToContract RLP-encoded transaction that creates the output for the contract
+     * @param outputTxToContractInclusionProof Second Transaction's inclusion proof
+     * @param rlpInputCreationTx RLP-encoded first transaction that transfers to this contract
+     * @param inputCreationTxInclusionProof First transactions inclusion proofs
+     * @param utxoPosInput position of the output that created the inputs for second transaction
+    */
+    function startExit(
+        uint256 utxoPosToExit,
+        bytes memory rlpOutputTxToContract,
+        bytes memory outputTxToContractInclusionProof,
+        bytes memory rlpInputCreationTx,
+        bytes memory inputCreationTxInclusionProof,
+        uint256 utxoPosInput
+    ) public payable {
+
+        verifyOwnership(rlpInputCreationTx, utxoPosInput);
+
+        PaymentTransactionModel.Transaction memory decodedSecondTx
+        = PaymentTransactionModel.decode(rlpOutputTxToContract);
+        require(
+            decodedSecondTx.inputs[0] == bytes32(utxoPosInput),
+            "Wrong utxoPosInput provided"
+        );
+
+        verifyTxValidity(
+            utxoPosInput,
+            rlpInputCreationTx,
+            inputCreationTxInclusionProof
+        );
+        runExit(
```

Comment:
same here  - let's assert that we actually did start an exit

Reply:
done `8d760f9`